sktime/classification/interval_based/_risf.py

```diff
@@ -0,0 +1,271 @@
+# -*- coding: utf-8 -*-
+"""Random Interval Spectral Ensemble (RISE)."""
+
+__author__ = ["Yi-Xuan Xu"]
+
+__all__ = ["RandomIntervalSpectralForest"]
+
+import numpy as np
+from deprecated.sphinx import deprecated
+from joblib import Parallel, delayed
+from sklearn.ensemble._base import _partition_estimators
+from sklearn.ensemble._forest import ForestClassifier
+from sklearn.tree import DecisionTreeClassifier
+from sklearn.utils.multiclass import class_distribution
+from sklearn.utils.validation import check_random_state
+
+from sktime.classification.base import BaseClassifier
+from sktime.classification.interval_based._rise import (
+    _make_estimator,
+    _parallel_build_trees,
+    _predict_proba_for_estimator,
+    _select_interval,
+)
+from sktime.utils.validation.panel import check_X, check_X_y
+
+
+@deprecated(
+    version="0.8.1",
+    reason="RandomIntervalSpectralForest will be moved or removed in v0.10.0, "
+    "to be replaced by the correctly named RandomIntervalSpectralEnsemble",
+    category=FutureWarning,
+)
```

Comment:
You don't need to keep this if it has been moved here?

Reply:
it does not conform to the new interface, so unless someone want to do that, I will delete it. I'll contact the code owner